apps/src/applab/applab.js

```diff
@@ -937,7 +937,7 @@ var displayFeedback = function () {
       executionError: Applab.executionError,
       response: Applab.response,
       level: level,
-      showingSharing: level.freePlay,
+      showingSharing: false,
```

Comment:
Did we previously (before your recent PR) just not use showingSharing for anything in App Lab?

Reply:
I'm not sure...

@poorvasingal, confirm that we *never* want to show sharing options in the "Finish" dialog for any App Lab level?

Reply:
@joshlory @poorvasingal confirmed that is correct - no share in "Finish" for any App Lab or Game Lab

Reply:
Note that this change will only affect App Lab.  Per https://codedotorg.slack.com/archives/C0T0GC9GE/p1515023020000162 this doesn't currently repro in Game Lab.  We could take a preventative fix for Game Lab here if you want?

https://github.com/code-dot-org/code-dot-org/blob/2ae2985b86384eed0cd233584aa7402a671efa82/apps/src/gamelab/GameLab.js#L1118